.ci_scripts/addX509User.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+echo
+echo "adding x509 Certificate subject as a User"
+# https://docs.mongodb.com/manual/tutorial/configure-x509-client-authentication/#add-x-509-certificate-subject-as-a-user
+
+PATH="$1"
```

Comment:
`PATH` not need if this one is `source`d as "sub-script"

Reply:
:+1: